junit5-jupiter-starter-maven/pom-SNAPSHOT.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
IIRC, this `http` usage is for the namespace reference only. Please revert it to `http`.

Reply:
The URL on the left is actually the "name" of the namespace.

See the `targetNampespace` attribute in the documented referenced by the URL on the right.

The URL on the right will be used by tools (or humans) to download the XSD.

Thus, `https` is correct for the URL on the right.